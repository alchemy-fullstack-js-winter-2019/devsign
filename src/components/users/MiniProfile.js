import React from 'react';

export default function MiniProfile() {
  return (
    <aside>
      <img src="../assets/prof-pic.png" width="30px" alt="prof pic"/>
      <h3>User Name</h3>
      <hr />
      <p>Chirps: 593</p>
      <p>Followers: 388</p>
    </aside>
  );
}
