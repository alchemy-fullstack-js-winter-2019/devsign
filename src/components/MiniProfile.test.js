import React from 'react';
import renderer from 'react-test-renderer';
import MiniProfile from './MiniProfile';

describe('MiniProfile', () => {
  it('matches a snapshot', () => {
    const tree = renderer.create(
      <MiniProfile />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
