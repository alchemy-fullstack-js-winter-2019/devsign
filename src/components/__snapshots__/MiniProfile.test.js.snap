// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MiniProfile matches a snapshot 1`] = `
<aside>
  <img
    src="../assets/prof-pic.png"
    width="30px"
  />
  <h3>
    User Name
  </h3>
  <hr />
  <p>
    Chirps: 593
  </p>
  <p>
    Followers: 388
  </p>
</aside>
`;
